hack/verify-shellcheck.sh

```diff
@@ -21,7 +21,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. && pwd -P)"
```

Comment:
We already use this pattern in other scripts, but not all.
`pwd -P` resolves symlinks. This is a portable way to fully resolve the repo root that only depends on:
- POSIX
-`${BASH_SOURCE[0]}`
- The level of nesting from the script to the root (`/..`)